cl/users/templates/emails/new_or_updated_webhook.txt

```diff
@@ -0,0 +1,7 @@
+Hello Admins!
+
+The following webhook was created or updated in Courtlistener:
+
+Webhook ID: {{ webhook.pk }}
+Webhook URL: {{ webhook.url }}
+Webhook status: {{ webhook.enabled }}
```

Comment:
Great, I added the links and the notification HTML version so it would be easier to only click the links.
I think the webhook endpoint is also useful so at first glance you could see if there is anything weird with the webhook.